src/OperationalInsights/OperationalInsights/Models/DataSourcePropertiesPerKind/PSCustomLogDataSourceProperties.cs

```diff
@@ -163,7 +163,12 @@ public class DateTimeExtraction
         /// <summary>
         /// Gets or sets the regex.
         /// </summary>
-        public string regex { get; set; }
+        public RegexDelimiter[] regex { get; set; }
+
+        ///<summary>
+        /// Gets or sets the FormatString
+        /// </summary>
+        public string formatString { get; set; }
```

Comment:
Also, why is this camelCase?  The appropriate casing for .Net properties is PascalCase

Reply:
aligning with the contract the backend expects.

Reply:
This is irrelevant.  You *must* use the appropriate .Net conventions, which means that properties are `PascalCase`

Reply:
sure, i will allign with .net conventions